css-view-transitions-2/Overview.bs

```diff
@@ -736,6 +747,10 @@ Prepend this to the [=Perform pending transition operations=] algorithm given a
 
 		1. [=Assert=]: |document| is [=fully active=].
 
+		1. [=Assert=]: |document| [=has been revealed=] is true.
```

Comment:
Do we still need to export this algorithm? It looks like for the new Document we're calling it within this spec in "resolve cross-document view-transition" and html should now use the `can initiate outbound view transition` bit.

Reply:
Export which algorithm?

Reply:
"resolve cross-document view-transition" shouldn't need to be visible outside this spec now. Can fix after the html rewrite lands.

Reply:
It's still called from page reveal

Reply:
What we don't need is "Resolve @view-transition rule", it's already not exported.

Reply:
I renamed it, I think it's clearer now